src/__tests__/labs/ViewLab.test.tsx

```diff
@@ -66,160 +66,149 @@ describe('View Lab', () => {
       },
     } as any)
 
-    let wrapper: any
-    await act(async () => {
-      wrapper = await mount(
-        <ButtonBarProvider.ButtonBarProvider>
-          <Provider store={store}>
-            <Router history={history}>
-              <Route path="/labs/:id">
-                <titleUtil.TitleProvider>
-                  <ViewLab />
-                </titleUtil.TitleProvider>
-              </Route>
-            </Router>
-          </Provider>
-        </ButtonBarProvider.ButtonBarProvider>,
-      )
-    })
-    wrapper.find(ViewLab).props().updateTitle = jest.fn()
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper }
+    return render(
+      <ButtonBarProvider.ButtonBarProvider>
+        <Provider store={store}>
+          <Router history={history}>
+            <Route path="/labs/:id">
+              <titleUtil.TitleProvider>
+                <ViewLab />
+              </titleUtil.TitleProvider>
+            </Route>
+          </Router>
+        </Provider>
+      </ButtonBarProvider.ButtonBarProvider>,
+    )
   }
 
-  describe('title', () => {
-    it('should have called the useUpdateTitle hook', async () => {
-      const expectedLab = { ...mockLab } as Lab
-      await setup(expectedLab, [Permissions.ViewLab])
-      expect(titleUtil.useUpdateTitle).toHaveBeenCalled()
-    })
-  })
-
   describe('page content', () => {
     it('should display the patient full name for the for', async () => {
```

Comment:
```suggestion
    it('should display the patients' full name', async () => {
```

Reply:
can you change the string to use " or ` so it can have ' inside it